template/src/app.test.ts

```diff
@@ -0,0 +1,16 @@
+import { screen } from "@testing-library/dom";
```

Comment:
using `screen` now as this is the preference for testing-library react, and reflects the side-effect nature of our tests now.